requirements.txt

```diff
@@ -5,7 +5,8 @@ envparse
 certifi
 elasticsearch-dsl>=5.0.0,<6.0.0
 networkx==1.9.1
-requests>=2.11.1,<3.0
```

Comment:
@afaulconbridge this version range was here for a reason. Did you check this is not a problem anymore?